src/app/components/typography/TypographyLabel.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <span :class="cn('text-xs font-semibold', classProps)">
```

Comment:
```suggestion
  <span :class="cn('text-[11px] font-semibold', classProps)">
```
`text-xs` is 12px but design specifies 11px, so in this case we must "hardcode" it.